cdap-watchdog/src/main/java/co/cask/cdap/metrics/store/DefaultMetricStore.java

```diff
@@ -113,6 +113,11 @@ public Cube get() {
       // i.e. for flows only
       ImmutableList.of(Constants.Metrics.Tag.NAMESPACE, Constants.Metrics.Tag.APP,
                        Constants.Metrics.Tag.FLOW)));
+    // queue
+    aggs.put("queue", new DefaultAggregation(
```

Comment:
FLOWLET_QUEUE in flow aggregation is still used for the system.process.events.out/in metrics. I am not sure whether it is ok to remove it.